open-company/open-company-web
# Private section no users

Card: https://trello.com/c/edD7WE0z
Item:
> No one left to invite to a private board https://projects.invisionapp.com/share/EBGDWDC25FD#/screens

To test:
- create a private section
- invite all the users to it
- select the invite input field again
- [x] do you see the prompt to add users to the team (as in the invision screen)? Good
- click on Carrot team Settings button
- [x] do you see the invite people panel appear? Good
